lib/interface/patientList/patientList.dart

```diff
@@ -124,7 +131,7 @@ class _MyAppState extends State<PatientList> implements PatientListContract {
                                   pat.sex == "Female"
                                       ? FontAwesomeIcons.female
                                       : FontAwesomeIcons.male,
-                                  color: Colors.grey,
+                                  color: Theme.of(context).disabledColor,
                                   size: 70.0,
```

Comment:
Here Too ⬆️

Reply:
I can't find a way to link the `size` with a meaningful Theme variable. Let me know if you can think of anything.

Reply:
I was just trying to say giving direct value in size might not work on bigger or smaller devices what I have learnt for responsiveness is to use MediaQuery and get height and width of the device and assign the ratio of your choice that fits well for particular thing🤔

Reply:
Like is said earlier, ignore the value of size but think about a way to control it. I should be able to manipulate its value using some universal parameters.
It just takes a minute to attain responsiveness so why worry about small stuff.
```
double width = MediaQuery.of(context).size.width;
double height = MediaQuery.of(context).size.height;
```

Reply:
Yea It's not a disastrous issue and we can add it anytime. I was just looking to learn is there any reason for doing that or not. Now I got that let's move into another issue 😉 

Thanks 😄